### PR TITLE
docs: add egghead lesson to quickstart

### PR DESCRIPTION
**Pull request by jlengstorf**
@calcsam had a good idea to add video versions of some of our commonly visited docs pages.

This adds a video walkthrough following the quick start from our docs to the quick start itself.

### CODE REVIEW

**Comment by jlengstorf** [comment]
Still waiting on approval from egghead. Once we have that plus the 👍 from @marcysutton, it should be good to go.

**Comment by jlengstorf** [comment]
Oh, wait — this one is approved. It's #11700 that needs approval from egghead.

So we just need @marcysutton on this one.

File: docs/docs/quick-start.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ This quick start is intended for intermediate to advanced developers. For a gent
 
 ## Use the Gatsby CLI
 
+<iframe class="egghead-video" width=600 height=348 src="https://egghead.io/lessons/gatsby-quick-start-with-gatsby-create-develop-and-build-gatsby-sites-from-the-command-line/embed" />
```
Comment (marcysutton) [inline docs/docs/quick-start.md]: Add a `title` to this iframe and it's good to go!

Comment (jlengstorf) [reply docs/docs/quick-start.md]: Done!

**Comment by jlengstorf** [comment]
@marcysutton title added! (I added them to #11700 as well.)

Let me know if you need anything else.

**Pull request merged**